draft-ietf-quic-http.md

```diff
@@ -1702,24 +1702,36 @@ extension is disabled if the setting is omitted.
 # Security Considerations
 
 The security considerations of HTTP/3 should be comparable to those of HTTP/2
-with TLS.  Note that where HTTP/2 employs PADDING frames and Padding fields in
-other frames to make a connection more resistant to traffic analysis, HTTP/3 can
-either rely on transport-layer padding or employ the reserved frame and stream
-types discussed in {{frame-grease}} and {{stream-grease}}.
+with TLS; the considerations from Section 10 of {{!HTTP2}} apply in addition to
+those listed here.
 
 When HTTP Alternative Services is used for discovery for HTTP/3 endpoints, the
 security considerations of {{!ALTSVC}} also apply.
 
+## Traffic Analysis
+
+Where HTTP/2 employs PADDING frames and Padding fields in other frames to make a
+connection more resistant to traffic analysis, HTTP/3 can either rely on
+transport-layer padding or employ the reserved frame and stream types discussed
+in {{frame-grease}} and {{stream-grease}}.  This provides slightly different
+granularities of padding and may involve different control surfaces.
```

Comment:
These methods of padding produce different results in terms of the granularity of padding, the effect of packet loss and recovery, and how an implementation might control padding.